### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by Mobinshahidi** [thedaviddias/Front-End-Checklist#528] [Q&A]
I want to know if we can translate this project to our native language?

**Comment by Harshithss07** [comment]
Fork the Repository , this  makes your own copy of the project. Then translate the Content ,it  includes:
README.md, UI text or labels, Documentation files.
Credit the Original Author – Mention in your translated README that it’s based on the original project.
also make a Pull Request  If the owner supports localization, you can submit your translation as a contribution.
